Formula/asuka.rb

```diff
@@ -32,7 +33,7 @@ def install
   test do
     input, _, wait_thr = Open3.popen2 "script -q screenlog.txt"
     input.puts "stty rows 80 cols 43"
-    input.puts "env LC_CTYPE=en_US.UTF-8 LANG=en_US.UTF-8 TERM=xterm #{bin}/asuka"
+    input.puts "env LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 TERM=xterm #{bin}/asuka"
```

Comment:
Fixes the test on Big Sur, breaks the test on Catalina 🙈

Reply:
Interesting.  It is on my list of formulae to investigate for Big Sur issues, but since 0.8.2 looks like it's coming I figured I'd wait on that